### PR TITLE
Edit Profile fixes

### PR DESCRIPTION
**Pull request by captainamiedi**
Fixes #

Changes in this pull request:
-
-
-

@sagararyal

### CODE REVIEW

**Comment by norbertschuler** [comment]
@captainamiedi can you link the issue which is fixed by this or add a small description was this is fixing?

File: src/features/user/UserProfile/components/AddTargetModal.tsx [hunk 1]
```diff
@@ -67,9 +69,14 @@ export default function AddTargetModal({
               defaultValue={userprofile.score.target ? userprofile.score.target : null}
               onChange={(e) => setTarget(e.target.value)}
               variant="outlined"
+              inputRef={register({
+                min: 0
```
Comment (norbertschuler) [inline src/features/user/UserProfile/components/AddTargetModal.tsx:73]: The backend declines zero as a possible target - so either the backend or the client should be changed:

Bildschirmfoto 2020-11-26 um 13 36 48

File: src/features/user/UserProfile/components/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from 'react'
```
Comment (norbertschuler) [inline src/features/user/UserProfile/components/Settings.tsx:1]: You should use an editor which can automatically nicely format our source code as in the file before.

File: src/features/user/UserProfile/components/AddTargetModal.tsx (hunk 1 above)
Comment (norbertschuler) [reply src/features/user/UserProfile/components/AddTargetModal.tsx:73]: Another issue: When I enter -100 as a value, I get this view with the string "invalid" after I clicked on the save button before the dialog closed automatically after the API call failed and my target does not get changed: 
Bildschirmfoto 2020-11-26 um 13 35 06

Comment (norbertschuler) [reply src/features/user/UserProfile/components/AddTargetModal.tsx:73]: I would suggest to show the user an error message if the API call fails and the target can not be changed.

**Review by norbertschuler — CHANGES_REQUESTED** [review]

**Comment by norbertschuler** [comment]
Tested again, but this still don't work for me (latest MacOS with latest Chrome). I neither see any loader when clicking on the save button for the new target value nor do I get an error message if I enter 0 or -100 as a value. Also the arrows allow me to decrease the value down to 0 instead only down to 1.

**Review by norbertschuler — APPROVED** [review]
LGFM

File: src/features/user/UserProfile/screens/IndividualProfile.tsx [hunk 3]
```diff
@@ -113,11 +115,11 @@ export default function IndividualProfile({ userprofile, changeForceReload,
...
-        variant="filled"
-        onClose={handleTextCopiedSnackbarClose} 
-        severity="success"
+        <MuiAlert
+          elevation={6}
+          variant="filled"
+          onClose={handleTextCopiedSnackbarClose}
+          severity="success"
         >
           Text Copied to Clipboard!
```
Comment (sagararyal) [inline src/features/user/UserProfile/screens/IndividualProfile.tsx:124]: @captainamiedi should this be translated?
@harshvitra do we still plan to use this alerts?

**Review by sagararyal — COMMENTED** [review]
.

File: src/features/user/UserProfile/screens/IndividualProfile.tsx (hunk 3 above)
Comment (captainamiedi) [reply src/features/user/UserProfile/screens/IndividualProfile.tsx:124]: These has been replaced with the share functionality.

Comment (sagararyal) [reply src/features/user/UserProfile/screens/IndividualProfile.tsx:124]: @captainamiedi Can you clean up the code?

Comment (captainamiedi) [reply src/features/user/UserProfile/screens/IndividualProfile.tsx:124]: ok

**Pull request merged**